key_test.go

```diff
@@ -0,0 +1,116 @@
+package shell
+
+import (
+	"context"
+	"testing"
+
+	"github.com/cheekybits/is"
+)
+
+func TestKeyGen(t *testing.T) {
+	is := is.New(t)
+	s := NewShell(shellUrl)
+
+	defer func() {
+		_, err := s.KeyRm(context.Background(), "testKey1")
+		is.Nil(err)
+	}()
+	key1, err := s.KeyGen(context.Background(), "testKey1", KeyGen.Type("ed25519"), KeyGen.Size(2048))
```

Comment:
```suggestion
	key1, err := s.KeyGen(context.Background(), "testKey1", KeyGen.Type("ed25519"))
```

per https://github.com/ipfs/go-ipfs-api/pull/220#discussion_r450294770 having ed25519 + 2048 bits does not make sense. Could you please remove the Size option here.